README.md

```diff
@@ -1,8 +1,64 @@
-# Test Repo
+# EnChord
 
-The chopsticks are the best female developer duo.
+**EnChord** is a perfect tool to develop your music dictation skills. Challenge yourself with different levels of quizzes and follow your progress by checking your highest score and average test results. You can also review what are the frequently mistaken interval types.
 
-## Scripts
+## Table of contents
+
+1. [How it works](https://github.com/TheChopsticks/EnChord/edit/main/README.md#how-it-works)
+2. [Future feature](https://github.com/TheChopsticks/EnChord/edit/main/README.md#future-features)
+3. [How EnChord was built](https://github.com/TheChopsticks/EnChord/edit/main/README.md#how-enchord-was-built)
+4. [Installation](https://github.com/TheChopsticks/EnChord/edit/main/README.md#installation)
+5. [Credits](https://github.com/TheChopsticks/EnChord/edit/main/README.md#credits)
+6. [License](https://github.com/TheChopsticks/EnChord/edit/main/README.md#license)
+7. [Badges](https://github.com/TheChopsticks/EnChord/edit/main/README.md#badges)
+
+## How it works
+
+#### Try 3 different levels
```

Comment:
I was wondering why we jumped from a level 2 (`##`) to a level 4 (`####`) heading here. Could we use a level 3 instead, or is it not a suitable choice here?

Reply:
I didn't really think it through here. I changed them now!

Reply:
I didn't really think it through here. I changed them now!